src/lkl/setup.c

```diff
@@ -1243,6 +1239,10 @@ static void* lkl_termination_thread(void* args)
                 "Could not unmount disk %d, %s\n", i, lkl_strerror(res));
         }
 
+#ifdef DEBUG
+    display_mount_table();
```

Comment:
Why is this moved?  Don't we want the mount table from before we unmount everything?

Reply:
Before the wrong merge, the idea was to output the mount table twice, before the unmount and after, to check that things have been correctly unmounted. The broken merge meant that we are outputting the mount table twice before the unmount, which  outputs the same information twice.

As an aside, I am not convinced that the unmouting always reliably flushes the file system state back to disk. I may open an issue on this.

Reply:
Ok, I didn't move it to the correct location, but I fixed this now.

I've also opened this issue: https://github.com/lsds/sgx-lkl/issues/754